components/gitpod-protocol/src/util/tracing.ts

```diff
@@ -39,7 +49,22 @@ export namespace TraceContext {
             "error": err.message,
             "stacktrace": err.stack
         })
-        ctx.span.setTag("error", true)
+        ctx.span.setTag("error", true);
+        if (errorCode) {
+            ctx.span.setTag("errorCode", errorCode);
+        }
+    }
+
+    export function logAPIError(ctx: TraceContext, err: ResponseError<any>) {
+        if (!ctx.span) {
+            return;
+        }
+        logError(ctx, err);
+
+        ctx.span.addTags({
+            errorCode: err.code,
+            apiError: true,
```

Comment:
`apiError` is quite generic, is there a namespeced tag we could use that might better convey what kind of error this is? If `apiError` is already used elsewhere then we'll just use it of course.

Reply:
Will rewrite to follow the jsonrpc convetion, than we have it namespaced. :+1: